ci-operator/step-registry/upi/conf/vsphere/ova/upi-conf-vsphere-ova-commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+HOME=/tmp
+export HOME
+
+echo "$(date -u --rfc-3339=seconds) - Locating RHCOS image for release..."
+ova_url="$(jq -r '.baseURI + .images["vmware"].path' /var/lib/openshift-install/rhcos.json)"
```

Comment:
Could this be parametrized? OKD installer uses `fcos.json` so it would be nice if it could be overriden

Reply:
@vrutkovs I think that - https://issues.redhat.com/browse/DPTP-1261
would fit better but of course not available at this point.
Maybe just a check that `fcos.json` exists if not use `rhcos.json`?

Reply:
Right, checking for `fcos.json` would solve this until DPTP-1261 is implemented (sorry, I though its already working)